build/broccoli/build-packages.js

```diff
@@ -98,7 +98,7 @@ function treeForPackage(packagePath) {
 
   let packageJSONTree = treeForPackageJSON(packagePath);
 
-  let tree = merge([packageTree, packageJSONTree]);
+  let tree = merge([packageTree, packageJSONTree, funnel(process.cwd(), { files: ['LICENSE'], destDir: '/' })]);
```

Comment:
We can't use process.cwd() as a tree (or we get recursive rebuilds depending on watchman).